1/1.4/Dockerfile

```diff
@@ -3,6 +3,8 @@ FROM gcr.io/google-appengine/debian9:latest
 # This is the release of Consul to pull in.
 ENV CONSUL_VERSION=1.4.4
 
+ENV C2D_RELEASE=${CONSUL_VERSION}
```

Comment:
Can we always have a static value assigned to the `C2D_RELEASE` variable, so we are sure what the result of the variable is.

For example: Currently if the `CONSUL_VERSION` will be modified to `1.4`, then we may not catch that `C2D_RELEASE` is incorrect during the review.

Reply:
As i remember we have agreed to use C2D_RELEASE to get version of app in v2 Makefile. 
Also we can write simple test in Makefile v2 to be sure C2D_RELEASE starts with TRACK and not match we ^TRACK$ 
What do you think?

Reply:
We can run tests, but maybe this is overkill as we already have smoke test in the Dockerfile.

I still would like to have static value here.